### PR TITLE
feat(migaloo): Add migaloo chain config

### PR DESCRIPTION
**Pull request by 0xFable**
Added Migaloo Chain config and 1 asset. Whale, the native token of the chain.

### CODE REVIEW

File: migaloo/chain.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  "codebase": {
+    "git_repo": "https://github.com/White-Whale-Defi-Platform/migaloo-chain",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": ["v1.0.0"],
+    "cosmos_sdk_version": "0.45",
+    "tendermint_version": "0.34",
+    "cosmwasm_version": "0.28",
+    "cosmwasm_enabled": true,
+    "genesis": {
+      "genesis_url": "https://github.com/White-Whale-Defi-Platform/migaloo-chain"
```
Comment (dylanschultzie) [inline migaloo/chain.json:41]: This'll probably need to be updated to:
```
https://raw.githubusercontent.com/White-Whale-Defi-Platform/migaloo-chain/main/networks/mainnet/genesis.json
```

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

File: migaloo/chain.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "migaloo",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.whitewhale.money/",
+  "pretty_name": "Migaloo",
+  "chain_id": "migaloo-1",
+  "bech32_prefix": "migaloo",
```
Comment (JeremyParish69) [inline migaloo/chain.json:9]: Please register this prefix to SLIP173: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: migaloo/chain.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      "genesis_url": "https://github.com/White-Whale-Defi-Platform/migaloo-chain"
+    }
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "20e1000e88125698264454a884812746c2eb4807",
+        "address": "seeds.lavenderfive.com:12856",
+        "provider": "Lavender.Five Nodes 🐝"
+      }
```
Comment (Galadrin) [inline migaloo/chain.json:50]:
```suggestion
      },
      {
        "id": "c936ae78abca1169362e068e3e94c87a0ace96c7",
        "address": "seeds.cros-nest.com:27656",
        "provider": "Cros-Nest"
      }
```

File: migaloo/chain.json [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+      {
+        "address": "https://whitewhale-rpc.polkachu.com",
+        "provider": "PolkaChu"
+      },
+      {
+        "address": "https://whitewhale-rpc.lavenderfive.com",
+        "provider": "Lavender.Five Nodes 🐝"
+      },
+      {
+        "address": "https://rpc.migaloo.silknodes.io",
```
Comment (Galadrin) [inline migaloo/chain.json:67]: Wron network 
"network":"narwhal-1"

**Review by Galadrin — CHANGES_REQUESTED** [review]

File: migaloo/chain.json (hunk 4 above)
Comment (0xFable) [reply migaloo/chain.json]: Thank you for that, removed !

File: migaloo/chain.json (hunk 3 above)
Comment (0xFable) [reply migaloo/chain.json:55]: Added this, thank you for including

File: migaloo/chain.json (hunk 2 above)
Comment (0xFable) [reply migaloo/chain.json:9]: Done Pr is here https://github.com/satoshilabs/slips/pull/1497 🤝

File: migaloo/chain.json [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  "$schema": "../chain.schema.json",
+  "chain_name": "migaloo",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.whitewhale.money/",
+  "pretty_name": "Migaloo",
+  "chain_id": "migaloo-1",
+  "bech32_prefix": "migaloo",
+  "daemon_name": "migalood",
+  "node_home": "$HOME/.migaloo",
```
[the remark this answers is not in the document]
Comment (0xFable) [reply migaloo/chain.json]: A great catch, updated this.

**Comment by 0xFable** [comment]
Pushed a commit dealing with most of these changes. Thanks for reviewing everyone. I was not able to request reviews from all so requested from one person. Open to any more feedback.

File: migaloo/chain.json [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+      "account_page": "https://explorer.silknodes.io/whitewhale/account/${accountAddress}"
+    },
+    {
+      "kind": "ping.pub",
+      "url": "https://ping.pub/migaloo",
+      "tx_page": "https://ping.pub/migaloo/tx/${txHash}"
+    },
+    {
+      "kind": "mintscan",
+      "url": "https://www.mintscan.io/migaloo",
```
[the remark this answers is not in the document]
Comment (0xFable) [reply migaloo/chain.json:103]: Not yet available. There is a PR open on the cosmosstation repo but waiting for merge there.

File: migaloo/chain.json (hunk 2 above)
Comment (0xFable) [reply migaloo/chain.json:9]: Following up, this pr is now merged https://github.com/satoshilabs/slips/pull/1497

File: migaloo/chain.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (0xFable) [reply migaloo/chain.json]: I removed this for now, will do another PR to add back whenever its live 👍

**Review by Galadrin — APPROVED** [review]

**Review by dylanschultzie — APPROVED** [review]

**Pull request merged**